src/complexity.cc

```diff
@@ -182,12 +182,17 @@ std::vector<BenchmarkReporter::Run> ComputeBigO(
     result_cpu = MinimalLeastSq(n, cpu_time, reports[0].complexity);
     result_real = MinimalLeastSq(n, real_time, result_cpu.complexity);
   }
-  std::string benchmark_name =
-      reports[0].benchmark_name.substr(0, reports[0].benchmark_name.find('/'));
+  std::string benchmark_name = reports[0].base_name;
```

Comment:
Hm, this looks like a non-NFC change, but i don't see anything in test changes?

Reply:
Base name is equivalent to everything leading up to the first `/`, so there's no reason to run a `substr`.